conformance_automated = "Fail"
errors_diff = """
Line 16: Unexpected errors ['generics_typevartuple_overloads.py:16:13: error[invalid-argument-type] `@Todo` is not a valid argument to `Generic`']
"""
output = """
generics_typevartuple_overloads.py:16:13: error[invalid-argument-type] `@Todo` is not a valid argument to `Generic`
"""
